.github/workflows/generate-docs.yml

```diff
@@ -0,0 +1,128 @@
+              echo -n '.'
+              sleep 5
+          done
+          echo ''
+          echo "::endgroup::"
+
+          echo "::group::Installing WordPress"
+          docker-compose exec -T -e MYSQL_PWD="$MYSQL_PWD" mysql mysql -e "CREATE DATABASE IF NOT EXISTS $WORDPRESS_DB_NAME"
+          docker-compose exec -T cli wp core install \
+            --title="Docs" --admin_user=admin --admin_password=password --admin_email=test@test.com --skip-email --url=http://localhost:$HOST_PORT
+          echo "::endgroup::"
+
+          echo "::group::Activate AMP plugin"
+          # Building the JS assets is required for the plugin to activate successfully, but since we're only generating
+          # the docs we simply create the file being checked for to save some time.
+          touch ../../assets/js/amp-block-editor.js
+          docker-compose exec -T cli wp plugin activate amp
+          echo "::endgroup::"
+        working-directory: bin/local-env
+
+      - name: Configure git user
+        run: |
+          git config user.email "pierregordon@protonmail.com"
+          git config user.name "Pierre Gordon"
+
+      - name: Check if remote branch exists
+        id: remote-branch
+        run: echo ::set-output name=exists::$([[ -z $(git ls-remote --heads origin "$HEAD_BRANCH" ) ]] && echo "0" || echo "1")
+        env:
+          HEAD_BRANCH: ${{ steps.branches.outputs.head }}
+
+      - name: Create branch to base pull request on
+        if: steps.remote-branch.outputs.exists == 0
+        run: git checkout -b "$HEAD_BRANCH"
+        env:
+          HEAD_BRANCH: ${{ steps.branches.outputs.head }}
+
+      - name: Fetch existing branch to add commits to
+        if: steps.remote-branch.outputs.exists == 1
+        run: |
+          git checkout "$HEAD_BRANCH"
+          git merge --no-edit "$BASE_BRANCH"
+        env:
+          BASE_BRANCH: ${{ steps.branches.outputs.base }}
+          HEAD_BRANCH: ${{ steps.branches.outputs.head }}
+
+      - name: Generate plugin documentation
+        run: docker-compose exec -T -u $(id --user) cli wp amp docs generate
+        working-directory: bin/local-env
+
+      - name: Check if there are changes
+        id: changes
+        run: echo ::set-output name=changed::$([[ -z $(git status --porcelain) ]] && echo "0" || echo "1")
+
+      - name: Commit and push changes
+        if: steps.changes.outputs.changed == 1
+        run: |
+          git add --all .
+          git commit -m "Update documentation"
+          git push origin "$HEAD_BRANCH"
+        env:
+          HEAD_BRANCH: ${{ steps.branches.outputs.head }}
+
+      - name: Create pull request
+        if: steps.changes.outputs.changed == 1 && steps.remote-branch.outputs.exists == 0
+        run: |
+          git push -u origin "$HEAD_BRANCH"
+          gh pr create --base "$BASE_BRANCH" --title "Update documentation for \`$BASE_BRANCH\` branch" --body "" --label Documentation
```

Comment:
Do the backticks not get evaluated? I guess not as seen in https://github.com/pierlon/amp-wp/pull/12.

Reply:
Nope.